fix: [BE] 임베디드 데이터베이스 sqlite3 추가

youngxpepp opened the pull request:
## 📕 제목

- 이슈 #47 
- 윈도우 환경에서 better-sqlite3 실행이 안되는 버그가 발생하여 조치함
- 프로덕션 환경에서 서버가 실행될 때 dist 디렉토리 내부 엔티티를 못 찾는 현상이 발생해서 조치함
- 윈도우 환경엥서 npm install 시에 better-sqlite3가 인스톨 안되는 경우 발생

## 📗 작업 내용

> 구현 내용 및 작업 했던 내역

- [x] sqlite3 추가
- [x] ~~better-sqlite3와 sqlite3를 선택할 수 있도록 구현~~
- [x] 프로덕션 환경에서 엔티티 경로를 찾지 못하는 버그 함께 수정
- [x] better-sqlite3 삭제

## 📘 PR 특이 사항

> PR을 볼 때 주의깊게 봐야하거나 말하고 싶은 점

- 윈도우에서 실행이 되나 확인해주세요.
- 환경변수 설정이 조금 바뀌었으니 위키를 다시 참조해주세요.

youngxpepp committed:
fix: [BE] 임베디드 데이터베이스 sqlite3 추가

- 기존 better-sqlite3가 윈도우 운영체제에서 실행 안되는 문제가 발생하여 sqlite3 추가
- DatabaseType에 SQLite3와 better-sqlite3를 구분함

youngxpepp committed:
fix: [BE] 엔티티 경로 수정
- 빌드된 파일들은 dist에 쌓이므로 프로덕션에서 실행하면 엔티티를 못 찾는 문제가 발생함
- 상대경로로 model을 찾도록 수정

Do-ho reviewed (approved):
LGTM! 좋은 PR 감사합니다 :)

jinhyukoo reviewed (approved):
건홍님 덕에 로컬에서 편하게 개발할 수 있을 것 같습니다! 감사합니다!

wooojini reviewed (approved):
sqlite3를 빠르게 반영해주셨군요!! 고생하셨습니다!!

youngxpepp committed:
fix: better-sqlite3 삭제

- 윈도우에서 npm install이 안되는 경우가 있어서 better-sqlite3 삭제
- 환경변수 DATABASE_TYPE에서 better-sqlite3 삭제
